examples/ruby/function.yaml

```diff
@@ -6,29 +6,27 @@ metadata:
 spec:
   handler: test.run
   runtime: ruby2.4
-  lambda: |
+  function: |
     # Obtains the latest Kubeless release published 
-    class Kubelessfunction
-       def self.run(request)
-          require "net/https"
-          require "uri"
-          require "json"
-      
-          # Fetch release info
-          uri = URI.parse("https://api.github.com/repos/bitnami/kubeless/releases")
-          http = Net::HTTP.new(uri.host, uri.port)
-          request = Net::HTTP::Get.new(uri.request_uri)
-          http.use_ssl = true
-          http.verify_mode = OpenSSL::SSL::VERIFY_PEER
-          response = http.request(request)
-      
-          # Parse response
-          output = JSON.parse(response.body)
-      
-          # Create a Hash for output 
-          output_hash = { :version => output[0]['name'] }
-      
-          # Print the stuff (JSON)
-          puts JSON.pretty_generate(output_hash)
-       end
+    def run(request)
+      require "net/https"
+      require "uri"
+      require "json"
+
+      # Fetch release info
+      uri = URI.parse("https://api.github.com/repos/kubeless/kubeless/releases")
+      http = Net::HTTP.new(uri.host, uri.port)
+      request = Net::HTTP::Get.new(uri.request_uri)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_PEER
```

Comment:
Aren't these the default parameters for `Net::HTTP`?

Reply:
Yeah I guess this line is redundant, not sure about `use_ssl`

Reply:
tested it and `use_ssl` is required, without it this fails